src/data/memberships.js

```diff
@@ -4,70 +4,112 @@ export default {
     destinationID: 'room1',
     destinationType: 'room',
     members: [
-      {personID: 'user1', muted: null, inMeeting: null},
-      {personID: 'user2', muted: null, inMeeting: null},
-      {personID: 'user3', muted: null, inMeeting: null},
-      {personID: 'user4', muted: null, inMeeting: null},
+      {
```

Comment:
can you revert to the compact format of this list?

Reply:
Getting eslint error

![image](https://user-images.githubusercontent.com/8277264/112800575-7f3b4c80-9078-11eb-8aef-9ed571bd9304.png)

Reply:
How come linter isn't catching [this line](https://github.com/webex/components/pull/182/files#diff-202791f48524a3e98537bfa5bb91dbf5e6e849322b0b5abda6faafeea9f8e089R99)?